src/test/java/com/commercetools/sync/commons/asserts/statistics/AssertionsForStatistics.java

```diff
@@ -103,4 +104,16 @@ public static StateSyncStatisticsAssert assertThat(@Nullable final StateSyncStat
     public static TaxCategorySyncStatisticsAssert assertThat(@Nullable final TaxCategorySyncStatistics statistics) {
         return new TaxCategorySyncStatisticsAssert(statistics);
     }
+
+    /**
+     * Create assertion for {@link CustomObjectSyncStatistics}.
+     *
+     * @param statistics the actual value.
+     * @return the created assertion object.
+     */
+    @Nonnull
+    public static CustomObjectSyncStatisticsAssert assertThat(@Nullable final CustomObjectSyncStatistics statistics) {
+        return new CustomObjectSyncStatisticsAssert(statistics);
+    }
+
```

Comment:
remove this line

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/569/commits/a8c7ce4c770a85be1efb2f658ea00f9ee8c58393